search-commons/src/main/java/no/unit/nva/search2/resource/ResourceSearchQuery.java

```diff
@@ -222,8 +225,11 @@ private void assignStatusImpossibleWhiteList() {
 
     public static class ResourceParameterValidator extends ParameterValidator<ResourceParameter, ResourceSearchQuery> {
 
+        private final String currentHost;
+
         ResourceParameterValidator() {
             super(new ResourceSearchQuery());
+            currentHost = HTTPS + searchQuery.getNvaSearchApiUri().getHost();
```

Comment:
always dev environment for host?

Reply:
Always set at runtime by `ParameterValidator.fromRequestInfo(RequestInfo);`